.cargo/config.toml

```diff
@@ -1,3 +1,6 @@
+[build]
+rustflags = ["-D", "warnings"]
```

Comment:
I had a bunch of issues with tests failing after fairly long CI runs, so I added this to make `cargo check`/`cargo build` fail locally in the same way as CI will.